src/components/DataSourceForm/index.tsx

```diff
@@ -0,0 +1,20 @@
+import DataSourceSelection from "../DataSourceSelection";
```

Comment:
could we use `@` symbol here?

Reply:
Only @/components/DataSourceSelection

Need to do?

Reply:
yes please